site/learn/expert-advice/authentication/multi-factor-authentication.md

```diff
@@ -0,0 +1,341 @@
+---
+layout: advice
+title: Multi-factor Authentication for Developers
+description: What is multi-factor authentication and why is it important?
+author: Dan Moore
+image: advice/breached-password-detection/expert-advice-breached-password-detection.png
+category: Security
+date: 2021-02-05
+dateModified: 2021-02-05
+---
+
+As more of our lives move online, multi-factor authentication (MFA) becomes increasingly important as a way of keeping our accounts secure. As a user, you know you should enable MFA on any accounts containing valuable data or which you want to keep safe. 
+
+As a developer or software engineer, MFA may seem a bit mysterious. This article will cover:
+
+* What MFA is
+* Why it is important
+* What factors are available 
+* When you might consider requiring MFA
+
+At the end, you should have a good understanding of options for integrating MFA into your applications, and how to start doing so.
+
+## What is Multi Factor Authentication?
```

Comment:
What is Multi-Factor Authentication (MFA)?

This can get picked up by Google SERPS